docs/general/electronics.md

```diff
@@ -84,8 +84,11 @@ Different countries have different standards for mains wiring and the used colou
 | Neutral        | ![white wire] ![grey wire]  | ![blue wire]  | ![black wire]  | ![black wire] | ![white wire] | ![black wire] |
 | Ground / Earth | ![green wire] ![earth wire] | ![earth wire] | ![earth wire]  | ![green wire] | ![green wire] | ![earth wire] |
 
-#### AC Inlet Europe & UK 220V
-[AC Inlet](/assets/images/general/electronics/AC-Inlet-EU-220V.jpeg)
```

Comment:
Update on my previous speculations: you actually *can* add alt text to references, as it turns out. That's what the double set of brackets is for. Unfortunately, this is pretty bulky compared to the compact form:
```markdown
![alt text][reference]
```
An alternative is simply specifying the title tag, and modifying the preprocessor (the program that converts Markdown into HTML) so that it takes the `title` of a reference based image to be its alt text if no alt text is explicitly specified where the image is. i.e.:
```markdown
![reference]
[reference]: /path/to/image.png "This title will be used to populate the Alt text"
```
```markdown
![This has its own alt text][reference]
[reference]: /path/to/image.png "This title will be used only as the title"
```

Although at this point this is purely an informational comment. This has strayed quite far from just your specific changes here. :)